src/relay/op/tensor/transform.cc

```diff
@@ -504,7 +511,21 @@ bool ReshapeRel(const Array<Type>& types,
     return false;
   }
 
-  const auto* param = attrs.as<ReshapeAttrs>();
+  Array<IndexExpr> oshape;
+
+  if (param->newshape.empty()) {
```

Comment:
I have one more comment here regarding the type of param->newshape. Do we want to make it Expr as well, so that we don't have empty attribute for symbolic newshape? Also this aligns with existing dynamic operators(arange).